.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       uses: actions/setup-java@v4
       with:
         distribution: 'zulu'
-        java-version: 17
+        java-version: 21
```

Comment:
Since we run `:javadoc` as part of the release job (https://github.com/mockito/mockito/actions/runs/7453524063/job/20279319914#step:4:95) I think we can omit the bump here. I think that will avoid the build issues that you experience, yet still solve our JavaDoc troubles. Thanks again for taking a look!